### ISSUE TITLE
Return statement is missing

### ISSUE THREAD

**Issue by hmennen90** [nunomaduro/larastan#1077]
- Larastan Version: 1.0.2
- `--level` used: 5
- Pull request with failing test: None

### Description

Running Larastan with default neon File as it is in the Docs, a Method in a Laravel Package returns a Model Instance which is not recognized

### Laravel code where the issue was found

```php
    /**
     * @return CategoryFactory
     */
    protected static function newFactory(): CategoryFactory {
        return new CategoryFactory();
    }
```
<img width="1579" alt="Bildschirmfoto 2022-01-05 um 11 08 39" src="https://user-images.githubusercontent.com/77828605/148199870-7d730b6a-b76d-417f-8a3d-d0fd108baa58.png">

**Comment by szepeviktor** [comment]
Hello Hendrik! 👋 

Very strange.

**Comment by canvural** [comment]
Hi,

I'm not able to reproduce this.

Please try again 1.0.3 If it still happens try to send a failing test case or prepare a reproduction repo. thank you.

**Comment by hmennen90** [comment]
@canvural can confirm this bug is resolved in 1.0.3 :-)
